DTS-STN/Alpha-Site
# update signup-info page

# Description

Update signup-info page to use data from AEM

## Test Instructions

1. Run `yarn dev` in the terminal
2. Go to `/signup-info` page
3. Make sure the page content displays correctly, check both English and French content, compare with the dev site
4. Make sure images displays correctly, image should resize automatically when changing screen resolution

## Checklist
